.pre-commit-config.yaml

```diff
@@ -19,7 +19,7 @@ repos:
         name: Flake8
         description: Run flake8 checks on the code
         additional_dependencies: [flake8-annotations]
```

Comment:
You should include `flake8-import-order` here with `flake8-annotations`

Reply:
alright